src/main/java/com/okta/sdk/clients/UserGroupApiClient.java

```diff
@@ -49,6 +49,9 @@ public UserGroupApiClient(ApiClientConfiguration config) {
     }
 
     public List<UserGroup> getUserGroupsWithLimit(int limit) throws IOException {
+        if (limit == -1) {
```

Comment:
There is a default limit, which we use in the `UserApiClient`, but not here. I'll add a method that doesn't take a limit.